include/os/linux/kernel/linux/xattr_compat.h

```diff
@@ -174,7 +174,7 @@ static int								\
 fn(const struct xattr_handler *handler, struct dentry *dentry,		\
     const char *name, const void *buffer, size_t size, int flags)	\
 {									\
-	return (__ ## fn(dentry->d_inode, name, buffer, size, flags));	\
+	return (__ ## fn(NULL, dentry->d_inode, name, buffer, size, flags));\
```

Comment:
I would really not do this NULL pointer passing. I've carefully coded this upstream so that idmapped mounts always pass the `init_user_ns` around as an indicator for non-idmapped mounts.

Reply:
These cases the xattr_handler callback does not provide user ns argument so a NULL is passed to the underlying "__***" function to match the args list.

I will think more about this NULL stuff.